python/xorbits/_mars/dataframe/groupby/tests/test_groupby_execution.py

```diff
@@ -1593,3 +1593,38 @@ def test_gpu_groupby_size(data_type, chunked, as_index, sort, setup_gpu):
     else:
         actual = actual.sort_index()
         pd.testing.assert_series_equal(expected, actual)
+
+
+# TODO: support cuda
+# @support_cuda
+def test_groupby_agg_on_same_funcs(setup_gpu, gpu):
```

Comment:
Does This UT test on CPU only for now? If true, should here just use ``setup``?

Reply:
This UT can run on GPU, but will fail due to #428 .

My plan is to uncomment the `@support_cuda` annotation after #428 being merged.